geometry_msgs/Pose pose
float64 quality
float64 width
